src/app/login/login.component.ts

```diff
@@ -0,0 +1,83 @@
+import { Component, OnInit, Input, Output, EventEmitter } from '@angular/core';
+import { Router } from '@angular/router';
+import { UserService } from '../services/user.service';
+import { User } from '../user';
+
+@Component({
+  selector: 'app-login',
+  templateUrl: './login.component.html',
+  styleUrls: ['./login.component.scss']
+})
+export class LoginComponent implements OnInit {
+  @Input() user: User;
+  resp: string;
+  @Output() submitted = new EventEmitter<boolean>();
+  constructor(private userService: UserService, private router: Router) {
+    let submitBtn = document.getElementById('submit');
+    submitBtn.addEventListener('click', (e: Event) => this.submitRegistration());
+  }
+
+  submitRegistration(){
+    let newUser = new User();
+    newUser.email = document.getElementById('emailR').value;
+    newUser.password = document.getElementById('passwordR').value;
+    newUser.firstName = document.getElementById('firstname').value;
+    newUser.lastName = document.getElementById('lastname').value;
+    newUser.address = document.getElementById('address').value;
+    newUser.creditCardNumber = document.getElementById('creditcardnumber').value;
+    newUser.cvv = document.getElementById('ccv').value;
+    newUser.role = 'USER';
+    this.userService.insert(newUser).subscribe(
+      (response) => {
+        this.resp = response;
+        if (this.resp !== 'User Inserted'){
```

Comment:
careful with matching strings like this when receiving a response on the backend.